### PR TITLE
Jh/yearn vault

### PR DESCRIPTION
**Pull request by cjinghong**

### CODE REVIEW

File: shared/src/components/Product/productCopies.tsx [hunk 1]
```diff
@@ -439,4 +439,103 @@ export const productCopies: { [vault in VaultOptions]: ProductCopies } = {
...
   },
+  "ryvUSDC-ETH-P-THETA": {
+    title: "T-yUSDC-P-ETH",
+    subtitle: "ETH Put-Selling",
+    description:
+      "Generates yield by running an automated ETH put selling strategy.",
+    tags: ["PUT-SELLING"],
+    strategy: (
+      <>
+        This vault earns yield on its yvUSDC deposits by running an automated
```
Comment (kenchangh) [inline shared/src/components/Product/productCopies.tsx:450]: Let's add a link to yvUSDC vault page https://yearn.fi/invest/0x5f18C75AbDAe578b483E5F43f12a39cF75b973a9

File: shared/src/constants/constants.ts [hunk 2]
```diff
@@ -7,6 +7,25 @@ export const NETWORK_NAMES: Record<number, string> = {
...
 
+export const FullVaultList = [
+  "ryvUSDC-ETH-P-THETA",
+  "rETH-THETA",
+  "rBTC-THETA",
+  "rUSDC-ETH-P-THETA",
+  "rUSDC-BTC-P-THETA",
+] as const;
+export type VaultOptions = typeof FullVaultList[number];
+const ProdExcludeVault: VaultOptions[] = ["rUSDC-BTC-P-THETA"];
```
Comment (kenchangh) [inline shared/src/constants/constants.ts:18]: Let's exclude the vault from prod for now

Comment (cjinghong) [reply shared/src/constants/constants.ts]: Yep done

File: shared/src/components/Product/productCopies.tsx (hunk 1 above)
Comment (cjinghong) [reply shared/src/components/Product/productCopies.tsx]: Updated with latest copies from Vinnie

File: shared/src/constants/constants.ts (hunk 2 above)
Comment (cjinghong) [reply shared/src/constants/constants.ts]: Yep done

**Review by kenchangh — APPROVED** [review]

**Pull request merged**